src/pcap/pcap.go

```diff
@@ -92,8 +88,8 @@ func (opts *CaptureOptions) validate() error {
 		return err
 	}
 
-	if len(opts.Filter) > 1000 {
-		return fmt.Errorf("expected filter to be less than 1000 characters")
+	if len(opts.Filter) > maxFilterLength {
+		return fmt.Errorf("expected filter to be less than 5000 characters, received %d", len(opts.Filter))
```

Comment:
Don't hardcode the error string if the condition is based on a variable.
```suggestion
		return fmt.Errorf("expected filter to be at most %d characters, received %d", maxFilterLength, len(opts.Filter))
```

Reply:
done